pages/contests/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
```

Comment:
what's the difference between this page and `pages/index.tsx` ?

Reply:
by default the contest page is the homepage, so you don't need an extra page

Reply:
removed